.github/workflows/deploy.yml

```diff
@@ -20,5 +20,7 @@ jobs:
           docker-compose run \
             -e PAGES_REPO_NWO=${{ github.repository }} \
             -e JEKYLL_GITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }} \
+            -e ELASTICSEARCH_API_KEY=${{ secrets.ELASTICSEARCH_API_KEY}} \
```

Comment:
Rename `deploy.yml` to `deploy_master.yml`? Or, preferably, combine the two since they seem to be doing mostly the same thing, but in different environments.

Reply:
This is cleaned up in #404